functions/Save-DbaCommunitySoftware.ps1

```diff
@@ -24,6 +25,7 @@ function Save-DbaCommunitySoftware {
         * DarlingData: Erik Darling's stored procedures (https://www.erikdarlingdata.com)
         * SQLWATCH: SQL Server Monitoring Solution created by Marcin Gminski (https://sqlwatch.io/)
         * WhoIsActive: Adam Machanic's comprehensive activity monitoring stored procedure sp_WhoIsActive (https://github.com/amachanic/sp_whoisactive)
+        * DbaMultitool: John McCall's T-SQL scripts for the long haul: optimizing storage, on-the-fly documentation, and general administrative needs (https://dba-multitool.org)
```

Comment:
Maybe change the "t" in "tool" to upper case like in the validate set of the software parameter?